apps/files_sharing/src/views/FilesHeaderNoteToRecipient.vue

```diff
@@ -6,7 +6,7 @@
 	<NcNoteCard v-if="note.length > 0"
 		class="note-to-recipient"
 		type="info">
-		<p v-if="user" class="note-to-recipient__heading">
+		<p v-if="displayName" class="note-to-recipient__heading">
```

Comment:
overlooked it in the master PR, but why adding a new computed instead of just:

```suggestion
		<p v-if="user.displayName" class="note-to-recipient__heading">
```

Reply:
Felt cleaner :thinking: